pe-parser-library/src/parse.cpp

```diff
@@ -1440,6 +1441,11 @@ bool getSymbolTable(parsed_pe *p) {
     }
 
     // Read auxiliary symbol records
+    auto nextSymbolOffset =
+        offset +
+        static_cast<std::uint32_t>(sym.numberOfAuxSymbols * SYMTAB_RECORD_LEN);
```

Comment:
I think this should be:
```c++
offset + (static_cast<std::uint32_t>(sym.numberOfAuxSymbols) *
          static_cast<std::uint32_t>(SYMTAB_RECORD_LEN));
```
assuming that `offset` is a `uint32_t`. This is because `numberOfAuxSymbols` is a `uint8_t`, and `SYMTAB_RECORD_LEN` is a `uint16_t`, and so my guess is that the original multiplication result would be `uint16_t`, zero-extended to `uint32_t`.